SPEC.md

```diff
@@ -304,6 +304,18 @@ The test index makes use of symbolic language in describing connection and messa
     -> mtPROOF_PATH_REQ
     <- mtPROOF_PATH_RESPONSE
 
+## Performance
+
+### ZG-PERFORMANCE-001
+
+    The node behaves as expected under load from other peers.
+    1. Establish a node and synthetic peers.
+    2. Begin simulation.
+    3. Introspect node health and responsiveness through peers (latency, throughput). This could be done using `Ping`/`Pong` messages.
+    There can be different errors during testing: broken pipes to signal established but suddenly lost connections, 
+    InvalidData errors to indicate that connection couldn,t be established at all or timeout erros to indicate 
+    connections where expected data has not been received.
```

Comment:
I don't really understand this statement,
did you mean something like: "...connections where expecting data and it has not been received"?

Reply:
Yes, I've changed the sentence if it was unclear.